tools/telemetry-generator/src/handlers/executionTimeTestHandler.ts

```diff
@@ -18,8 +18,8 @@ module.exports = function handler(fileData, logger): void {
 			benchmarkType: "ExecutionTime",
 			suiteName: fileData.suiteName,
 			benchmarkName: testData.benchmarkName,
-			arithmeticMean: testData.stats.arithmeticMean,
-			marginOfError: testData.stats.marginOfError,
+			arithmeticMean: testData.customData["Heap Used Avg"],
```

Comment:
`arithmeticMean` should be swapped to `period (ns/op)`. 
`marginofError` is missing, thus the package should be updated to `0.50.0` now.

Reply:
The problems here are all resolved by separate PRs.

Reply:
Change `Heap Used Age` to `period (ns/op)`.

Reply:
Update. Thx